proto/opamp.proto

```diff
@@ -25,8 +25,8 @@ option go_package = "github.com/open-telemetry/opamp-go/protobufs";
 message AgentToServer {
     // Globally unique identifier of the running instance of the Agent. SHOULD remain
     // unchanged for the lifetime of the Agent process.
-    // Recommended format: https://github.com/ulid/spec
-    string instance_uid = 1;
+    // MUST be 16 bytes long and SHOULD be generated using UUID v7 spec.
```

Comment:
```suggestion
    // MUST be 16 bytes long and SHOULD be generated using the UUID v7 spec.
```
Small nit. Same for the other mentions of the UUID v7 spec.

Reply:
Fixed.